go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitrise-steplib/steps-go-test
 
-go 1.22
+go 1.21
```

Comment:
go 1.22 caused a step preparation error: `select an appropriate Go installation for compiling the Step: check go version: exit status 2`
https://app.bitrise.io/build/2e6302d2-2d23-409b-95e0-0c392134b85f?tab=log